addons/udes_stock/models/procurement.py

```diff
@@ -8,3 +8,44 @@ class ProcurementGroup(models.Model):
 
     # Allow groups to be created
     MSM_CREATE = True
+
+    def _get_orderpoint_domain(self, company_id=False):
+        """ Override order point domain to filter by active_ids
+        """
+        domain = super()._get_orderpoint_domain(company_id=company_id)
+        orderpoints = self.env.context.get('active_ids', None)
+        if orderpoints:
+            domain += [('id', 'in', orderpoints)]
+        return domain
+
+        
+class StockWarehouseOrderPoint(models.Model):
+    _inherit = 'stock.warehouse.orderpoint'
+
+    @api.model
+    def check_order_points(self, use_new_cursor=False, company_id=False):
+        """
+        Copy of run_scheduler from the stock module's ProcurementGroup class.
+        This allows us to only check order points.
+        """
+        OrderPoint = self.env["stock.warehouse.orderpoint"]
+        ProcurementGroup = self.env["procurement.group"]
+        try:
+            if use_new_cursor:
+                cr = registry(self._cr.dbname).cursor()
+                domain = ProcurementGroup._get_orderpoint_domain()
+                self = OrderPoint.with_context(prefetch_fields=False).search(domain)
+                self = self.with_env(self.env(cr=cr))
+
+            self.sudo()._procure_orderpoint_confirm(
+                use_new_cursor=use_new_cursor,
+                company_id=company_id)
+            self._cr.commit()
```

Comment:
I think we should be committing `cr` here, not `self._cr`? (once the guard on `use_new_cursor` is reinstated)

Reply:
Nope, I didn't noticed we'd redefined `self` :facepalm: